pkgs/development/python-modules/ilua/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+, jupyter_console
+, pygments
+, termcolor
+, txzmq }:
+
+buildPythonApplication rec {
+  pname = "ilua";
+  version = "0.2.1";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-YxV6xC7GS5NXyMPRZN9YIJxamgP2etwrZUAZjk5PjtU=";
+  };
+
+  propagatedBuildInputs = [
+    jupyter_console
+    pygments
+    termcolor
+    txzmq
+  ];
```

Comment:
Upstream is mentioning [`jupyter_core`](https://github.com/guysv/ilua/blob/master/setup.cfg#L38).

Reply:
Done